### ISSUE TITLE
Button is not accessible by keyboard (critical)

### ISSUE THREAD

**Issue by MelanieHotchkiss** [magda-io/magda#1908]
### Problem description
Intopia issue 15 - The feedback attachment button is not keyboard accessible.

### Resolution
We need to log a bug with Zendesk... probably better if it comes from someone at DTA who actually has an account, so assigning this to @maxious :)

### Screenshot / Design / File reference
[15] Button is not accessible by keyboard
Type: WCAG 2.1 failure
Fails 2.1.1 Keyboard - Level A
Location: Feedback widget, all pages
Platform: Desktop
Severity: Critical
What’s the issue?
intopia.digi tal 25
The feedback attachment button is not keyboard accessible.
Why it matters
The ability for a keyboard only user to upload a file in support of feedback is crucial to the value of the feedback tool.
Our recommendation
Remove style="display: none; " from the input markup to reveal the file selection option and make the attachment tool keyboard accessible.
Feedback widget with display:none
Feedback widget without display:none
intopia

**Comment by AlexGilleran** [comment]
Assigning this to you @maxious because it's Zendesk :).

**Comment by maxious** [comment]
Apparently they did a accessibility audit recently but must have missed this... https://support.zendesk.com/hc/en-us/articles/360001899447-Web-Widget-Accessibility-Improvements

Aside from disabling Zendesk, we could just disable file uploads https://developer.zendesk.com/embeddables/docs/widget/settings#attachments

**Comment by maxious** [comment]
Attachments disabled.
